README.md

```diff
@@ -37,8 +37,9 @@ The core approach taken is to strongly encourage and enforce separation of conce
 ### Building binary
 Run the following command to build the binary used in the following commands:
 ```shell
-go build --race
+go build
```

Comment:
I don't think there is a reason not to include `--race` in the example command, and have the note as well.

Reply:
There are performance costs from having race in there likely not too much of an issue in our case but they will be there. 
https://go.dev/doc/articles/race_detector#Runtime_Overheads